### ISSUE TITLE
choose a license for this example app

### ISSUE THREAD

**Issue by wilkolazki** [raymondfeng/loopback4-example-websocket#4]
Hi,

Your example application is a great bootstrap resource and one is tempted to copy-paste fragments of your code. This is, however, illegal until you grant your users some kind of license. 

Please choose some kind of open source license, possibly one that allows including in closed-source software, so others might copy-paste with clear conscience. 

Best regards,
Szymon Wilkołazki

**Comment by raymondfeng** [comment]
Good point. I'll update. Please use https://github.com/strongloop/loopback-next/tree/socketio instead.
